### PR TITLE
Norwegian: Completed!

### PR DESCRIPTION
**Pull request by DanielRTRD**
Full comparison with English file, updated accordingly. Including countries!

### CODE REVIEW

File: locales/nb/pagination.php [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are used by the paginator library to build
+    | the simple pagination links. You are free to change them to anything
+    | you want to customize your views to better match your application.
+    |
+    */
+
     'previous' => '&laquo; Forrige',
+    'next'     => 'Neste &raquo;',
```
Comment (caouecs) [inline locales/nb/pagination.php:16]: why set the order ?

File: locales/nb/validation-inline.php [hunk 2]
```diff
@@ -1,131 +1,143 @@
...
+    'before_or_equal'      => ':attribute må være en dato før eller lik :date.',
     'between'              => [
-        'array'   => 'This content must have between :min and :max items.',
-        'file'    => 'This file must be between :min and :max kilobytes.',
-        'numeric' => 'This value must be between :min and :max.',
-        'string'  => 'This string must be between :min and :max characters.',
+        'numeric' => ':attribute må være mellom :min - :max.',
+        'file'    => ':attribute må være mellom :min - :max kilobytes.',
+        'string'  => ':attribute må være mellom :min - :max tegn.',
+        'array'   => ':attribute må ha mellom :min - :max elementer.',
```
Comment (caouecs) [inline locales/nb/validation-inline.php:30]: sort

File: locales/nb/validation-inline.php [hunk 3]
```diff
@@ -1,131 +1,143 @@
...
-    'digits'               => 'This must be :digits digits.',
-    'digits_between'       => 'This must be between :min and :max digits.',
-    'dimensions'           => 'This image has invalid dimensions.',
-    'distinct'             => 'This field has a duplicate value.',
-    'email'                => 'This must be a valid email address.',
-    'ends_with'            => 'This must end with one of the following: :values.',
-    'exists'               => 'The selected value is invalid.',
-    'file'                 => 'The content must be a file.',
-    'filled'               => 'This field must have a value.',
+    'boolean'              => ':attribute må være sann eller usann',
```
Comment (caouecs) [inline locales/nb/validation-inline.php:32]: In validation-inline.php we don't have :attribute

File: locales/nb/validation.php [hunk 4]
```diff
@@ -20,14 +20,14 @@
...
+    'attached'             => ':attribute er allerede vedlagt.',
     'before'               => ':attribute må være en dato før :date.',
     'before_or_equal'      => ':attribute må være en dato før eller lik :date.',
     'between'              => [
-        'array'   => ':attribute må ha mellom :min - :max elementer.',
-        'file'    => ':attribute må være mellom :min - :max kilobytes.',
         'numeric' => ':attribute må være mellom :min - :max.',
+        'file'    => ':attribute må være mellom :min - :max kilobytes.',
         'string'  => ':attribute må være mellom :min - :max tegn.',
+        'array'   => ':attribute må ha mellom :min - :max elementer.',
```
Comment (caouecs) [inline locales/nb/validation.php:30]: sort

File: locales/nb/nb.json [hunk 5]
```diff
@@ -1,20 +1,20 @@
 {
+    ":days day trial": ":days dagers prøveversjon",
```
Comment (caouecs) [inline locales/nb/nb.json:2]: sort

**Review by caouecs — CHANGES_REQUESTED** [review]
Problem with validation-inline and :attribute

and don't set sort of tags

File: locales/nb/pagination.php (hunk 1 above)
Comment (DanielRTRD) [reply locales/nb/pagination.php:16]: Compared to and changed according to the English file.

File: locales/nb/validation-inline.php (hunk 2 above)
Comment (DanielRTRD) [reply locales/nb/validation-inline.php:30]: Same here, compared to and changed according to the English file.

File: locales/nb/validation-inline.php (hunk 3 above)
Comment (DanielRTRD) [reply locales/nb/validation-inline.php:32]: Will fix.

File: locales/nb/validation.php (hunk 4 above)
Comment (DanielRTRD) [reply locales/nb/validation.php:30]: Same here, compared to and changed according to the English file.

**Comment by caouecs** [comment]
Thank you, I will sort tags

**Comment by caouecs** [comment]
For country names, if it's same, we can add its in exclude file

**Comment by DanielRTRD** [comment]
> For country names, if it's same, we can add its in exclude file

Yes, I have checked everything.

**Comment by caouecs** [comment]
Ok, I add these tags ( see: https://laravel-lang.github.io/lang/statuses/nb.html ) in exclude file

**Pull request merged**